remote-config.jsonc

```
@@ -153,7 +153,10 @@
           "message": "DDEV has support for many database browsers. See them all at https://ddev.readthedocs.io/en/stable/users/usage/database-management/#database-guis"
         },
         {
-          "message": "Pro tip: `ddev config --auto` will update your project config to ensure it’s compatible with the version of DDEV that’s running."
+          "message": "Pro tip: `ddev config --auto` will update your project config to ensure it’s compatible with the version of DDEV that’s running, but it won't change configured values."
```

i wonder, in the current version as a naive user i would ask how is it possible that my config is updated and adjusted with the latest version while not changing any configured value. that would leave me behind with a big question mark. Would it make sense to write instead of "ensure that it is compatible"  express something like "ensure that the config file contains all the latest keys (*not sure about the right term here in the context of the config file) needed for running the latest version of ddev. existing already configured values won't be changed"

---

I'm not sure how we can solve this in one sentence.

---

how about something like that? 

```suggestion
          "message": "Pro tip: `ddev config --auto` will update your project config to ensure it contains all the keys available for the version of DDEV you are running. Already configured values remain untouched."
```

---

It doesn't make it contain all the keys available. Mostly it removes some in this situation.

---

but it is aligning the list of keys? (removing and adding?)

---

Reworded a bit.